tests/bootstrap.php

```diff
@@ -1,12 +1,17 @@
 <?php
 
-define('PHPUNIT_RUN', 1);
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
 
-require_once __DIR__ . '/../../../lib/base.php';
+declare(strict_types=1);
 
-\OC::$composerAutoloader->addPsr4('Test\\', OC::$SERVERROOT . '/tests/lib/', true);
-\OC::$composerAutoloader->addPsr4('Tests\\', OC::$SERVERROOT . '/tests/', true);
+use OCP\App\IAppManager;
+use OCP\Server;
 
-OC_App::loadApp('sharelisting');
+require_once __DIR__ . '/../../../lib/base.php';
+require_once __DIR__ . '/../../../tests/autoload.php';
+require_once __DIR__ . '/../vendor/autoload.php';
 
-OC_Hook::clear();
+Server::get(IAppManager::class)->loadApp('approval');
```

Comment:
```suggestion
Server::get(IAppManager::class)->loadApp('sharelisting');
```
The copy&paste went too far I think